tg_bot/__main__.py

```diff
@@ -415,6 +415,34 @@ def migrate_chats(bot: Bot, update: Update):
     LOGGER.info("Successfully migrated!")
     raise DispatcherHandlerStop
 
+def is_chat_allowed(bot, update):
+    if len(WHITELIST_CHATS) =! 0:
+        chat_id = update.effective_message.chat_id
+        if chat_id not in WHITELIST_CHATS:
+            bot.send_message(chat_id=update.message.chat_id,
+                             text='Unallowed chat! Leaving...')
+            bot.leave_chat(chat_id)
+        else:
+            pass
+    elif len(BLACKLIST_CHATS) != 0:
```

Comment:
by using an elif, you contradict the statement in the readme. You state that the blacklist takes priority, but this doesnt happen here, as the whitelist is evaluated first.